### PR TITLE
Fix #27: Replace every self.client occurences with self.airtable_client to avoid conflicts

### LINKED ISSUE #27: Rename self.client to something more explicit to express what the client is for.
We have `self.client` in this package, but if we have another package on a Wagtail Page or Snippet that also uses `self.client`, we are going to run into a conflict. 

I suggest finding and replacing `self.client` with `self.airtable_client` across all files.

### PR DESCRIPTION
**Pull request by fabienheureux**
Fixes https://github.com/wagtail/wagtail-airtable/issues/27

### CODE REVIEW

**Comment by KalobTaulien** [comment]
Amazing, thank you so much for doing this @fabienheureux!

File: tests/test_views.py [hunk 1]
```diff
@@ -8,36 +8,36 @@ class TestAdminViews(TestCase):
     fixtures = ['test.json']
 
     def setUp(self):
-        self.client.login(username='admin', password='password')
+        self.airtable_client.login(username='admin', password='password')
```
Comment (KalobTaulien) [inline tests/test_views.py:11]: In here we actually want to use `self.client` for tests. This is the test request client.

File: tests/test_views.py [hunk 2]
```diff
@@ -8,36 +8,36 @@ class TestAdminViews(TestCase):
...
 
     def test_get(self):
-        response = self.client.get('/admin/airtable-import/')
+        response = self.airtable_client.get('/admin/airtable-import/')
```
Comment (KalobTaulien) [inline tests/test_views.py:14]: Same here, but only for tests.

**Review by KalobTaulien — APPROVED** [review]

**Comment by KalobTaulien** [comment]
There were some test things that didn't need to be changed. That was my mistake when filing the issue. I'll merge this in and touch this up.

**Comment by fabienheureux** [comment]
Ah right, I mainly submitted this PR to get familiar with potential contribution guidelines in the repo / linting etc before going further with the other one I submitted. 

I did not put too much thinking into these changes unfortunately 😅

**Pull request merged**